Unix/installbuilder/conf/omiserver.conf

```diff
@@ -46,3 +46,14 @@ keyfile=/etc/opt/omi/ssl/omikey.pem
 configfile=/etc/opt/omi/conf/omiserver.conf
 #NoSSLv2=true
 #NoSSLv3=false
+# To enable NTLM authentication via negotiation (SPNEGO), specify the location 
+# of the credentials file. This file is in the form
+#
+# <domain>|<username>:<password> 
+#
+# For further information, take a look at the file Unix/doc/setup-ntlm-omi.md
```

Comment:
You're assuming that the reader is familiar with GitHub, or with the source tree. Most OMI customers won't be familiar with either. I recommend something like:

`# For further information, please see link: https://github.com/Microsoft/omi/blob/master/Unix/doc/setup-ntlm-omi.md`